bin/notifications.sh

```diff
@@ -195,7 +197,27 @@ function notification() {
 					log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "Email disabled and not creating file: ${_project_state_file}.mailed"
 					continue
 				else
-					sendEmail "${_project_state_file}" "${_project}" "${_run}" "${_phase}" "${_state}" "${_lfs_root_dir}" "${_controlFileBase}"
+					sendEmail='true'
+					IFS='/' read -r -a _traceSpecifications <<< "${_action}"
+					maxTime="${_traceSpecifications[1]:-0}" 
+					if [[ "${maxTime}" -ne '0' ]]
+					then
+						maxTimeMin=$((${maxTime}*60))
+						timeStampPipeline=$(find "${_project_state_file}" -mmin +"${maxTimeMin}")
+						if [[ -z "${timeStampPipeline}" ]]
+						then
+							log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "The file ${_project_state_file} is not available or not older than ${maxTime} hours"
+							sendEmail='false'
+						else
+							echo -e "Dear HPC helpdesk,\n\nPlease check if there is something wrong with the ${_phase}.\nThe ${_phase} for project ${_project} is not finished after ${maxTime} hours.\n\nKind regards\nHPC Helpdesk" > "${_project_state_file}.mail"
```

Comment:
`${_project_state_file}.mail` is not read anymore to put it in the body of the email message...